README.md

```diff
@@ -1,202 +1,77 @@
-# IoT Reference Integration on STM32U5 Arm Cortex-M33 MCU
+# Getting Started Guide for STM32U5 IoT Discovery Kit with AWS
 
+## Overview
 
-### Perfroming Over-the-air (OTA) Firmware Update for [B-U585I-IOT02A discovery kit](https://www.st.com/en/evaluation-tools/b-u585i-iot02a.html)
+This project demonstrates how to integrate modular FreeRTOS software with hardware enforced security to create secure and updatable cloud connected applications. The project is pre-configured to run on the STM32U585 IoT Discovery Kit and connect to AWS.
 
-The project shows an IoT reference implementation of how to integrate FreeRTOS libraries on STM32U5 platform to perform OTA update with AWS IoT using both trustzone and non-trustzone hardware capablities. The demo runs FreeRTOS OTA agent as one of the RTOS tasks in background, which waits for OTA updates from cloud. The non-trustzone version of the demo leverages internal flash memory's dual bank. The total 2MB internal flash is split into two banks of 1MB each. The main firmware is running on one bank, while downloading new firmware image into another bank. Thus, the total firmware image size should not exceed 1MB.
+The STM32U585 IoT Discovery kit comes with 2048 KB of Flash memory, 786 kB of RAM and is based on Arm Cortex®-M33.
 
-Please follow the below steps to bring up the demo project and perform OTA update:
+The STM32U5 IoT Discovery Kit is equipped with a Wi-Fi and Bluetooth module, microphones, a temperature and humidity sensor, a magnetometer, an accelerometer and gyroscope, a pressure sensor, as well as Time-of-Flight (ToF) and gesture-detection sensors.
 
-#### Pre-requisites
+The board also comes with 512-Mbit octal-SPI Flash memory, 64-Mbit octal-SPI PSRAM, 256-Kbit I2C EEPROM, as well as ARDUINO Uno V3, STMod+, and Pmod expansion connectors, plus an expansion connector for a camera module, and STLink-V3E embedded debugger.
 
-* AWS CLI
-  * The steps below uses AWS CLI commands to perform interactions with cloud,
-  * AWS CLI can be installed using this [link](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
-  * Run `aws configure` and set the access Key ID, secret Key and region for your AWS account
+The following project folder consists of a **non secure version(b_u585i_iot02a_ntz)** of the project and **secure version(b_u585i_iot02a_tfm)** of the project. The following shows the steps to follow for connecting the secure project to AWS and doing an OTA(Over the Air) Update and also connecting the non secure project to AWS. 
 
-* If not created already, provision a new thing name and have the required certificates and policies attached to connect to AWS IoT broker.
+## Hardware Description
```

Comment:
For the readme in the main page, please follow this structure:
Introduction
Folder Structure -> brief description of "Common", "Drivers", "Middleware", "Projects", "tools", etc...
Demos -> brief description of non trust zone and tfm projects
Cloning the Repository -> instruction how users should clone the repo
Running the Demos -> Start Getting Started Guide here with references to the two Readme you have.